selinux/ceph.te

```diff
@@ -106,7 +106,9 @@ allow ceph_t self:process setpgid;
 allow ceph_t var_run_t:dir { write create add_name };
 allow ceph_t var_run_t:file { write create open getattr };
 
+ifdef(`distro_redhat', `)
```

Comment:
There is a typo, here: the ')' character at the end of the line should not be there.